core/editor.js

```diff
@@ -524,21 +524,7 @@
 			// The list of URLs to language files.
 			var languageFiles = [];
 
-			/**
-			 * An object that contains references to all plugins used by this
```

Comment:
`editor.plugins` object docs are not visible with umberto built - not sure if they should, would like to know your opinion.

Reply:
Hmm, they appear as a description of `plugins` property in `editor` page. Seems like Umberto upstream issue.